From 2942500d2820c8922a869322fb0b00c412ef5cce Mon Sep 17 00:00:00 2001
From: Steve Lhomme <robux4@ycbcr.xyz>
Date: Tue, 13 Feb 2018 18:04:23 +0100
Subject: [PATCH] d3d11_fmt: assume drivers are bad on Winstore builds

Since we have no wait to get the version we fallback to the worst case.
---
 modules/video_chroma/d3d11_fmt.c | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/modules/video_chroma/d3d11_fmt.c b/modules/video_chroma/d3d11_fmt.c
index 5a321e3..12ff367 100644
--- a/modules/video_chroma/d3d11_fmt.c
+++ b/modules/video_chroma/d3d11_fmt.c
@@ -360,12 +360,12 @@ int D3D11CheckDriverVersion(d3d11_device_t *d3d_dev, UINT vendorId, const struct
     if (FAILED(hr))
         return VLC_EGENERIC;
 
-#if VLC_WINSTORE_APP
-    return VLC_SUCCESS;
-#else
     if (vendorId && adapterDesc.VendorId != vendorId)
         return VLC_SUCCESS;
 
+#if VLC_WINSTORE_APP
+    return VLC_EGENERIC;
+#else
     bool newer =
            d3d_dev->WDDM.wddm > min_ver->wddm ||
           (d3d_dev->WDDM.wddm == min_ver->wddm && (d3d_dev->WDDM.d3d_features > min_ver->d3d_features ||
-- 
2.10.1.windows.1

